shared/message/vision/Ball.proto

```diff
@@ -23,29 +23,22 @@ package message.vision;
 
 import "google/protobuf/timestamp.proto";
 import "Vector.proto";
-import "message/Geometry.proto";
 import "Matrix.proto";
 
 message Ball {
-    enum MeasurementType {
-        UNKNOWN      = 0;
-        REPROJECTION = 1;
-        WIDTH_BASED  = 2;
-    }
-    message Measurement {
-        MeasurementType type = 1;
-        /// Vector from camera to ball in camera coordinates
-        fvec3 srBCc      = 2;  // Spherical Reciprocal Coordinates (1/distance, phi, theta)
-        fmat3 covariance = 3;
-    }
-    /// Cone describing the ball from the camera's perspective
-    Cone                 cone         = 1;
-    repeated Measurement measurements = 2;
+    /// Central axis unit vector of the ball from the camera in camera space
+    fvec3 uBCc = 1;
+    // Spherical Reciprocal Coordinates (1/distance, phi, theta)
+    fvec3 srBCc = 2;
+    /// Unit radius of the ball from the camera in world space
+    /// Cosine of the angle from the centre of the ball to the edge of the ball
+    float radius = 2;
     /// Angular positions from the camera
-    fvec2 screen_angular = 3;
-
-    fvec2 angular_size = 4;
-    /// Colour of the ball for NUsight. Changes depending on the perceived ball location
+    fvec2 screen_angular_BCw = 4;
```

Comment:
Really not a fan of this name ... but also dont have any suggestions for improving it

Reply:
My suggestion was deleting it :P

Reply:
I deleted it...